rolling/distribution.yaml

```diff
@@ -3911,6 +3911,17 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
       version: master
     status: developed
+  rot_conv_lib:
```

Comment:
@ijnek nit: no `doc` repository?

Reply:
This is something I'd like to clarify in the documentation.
Are we required to add a ``doc`` entry? If there are no docs, is it fine to just skip it?

Reply:
> Are we required to add a doc entry?

You're not required, no. 

> If there are no docs, is it fine to just skip it?

Yes, but do note that documentation could be as little as a README. 

---

Anywho, I'm just checking it wasn't an unintended omission.

Reply:
I thought the ``doc`` field was specifically for ROS docs.

For example, ``ackermann_msgs`` [doesn't have a doc entry](https://github.com/ros/rosdistro/blob/a9217da23e9fe13875966df7a140bad251edfb23/rolling/distribution.yaml#L42), but the [README is displayed](https://index.ros.org/r/ackermann_msgs/) on ROS index correctly.

The omission is intended, but I'd like to clarify this so I can clarify it in https://github.com/ros2/ros2_documentation/pull/2375